cypress/support/index.js

```diff
@@ -18,3 +18,57 @@ import './commands'
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+
+// https://github.com/cypress-io/cypress/issues/1313#issuecomment-449456550
+// eslint-disable-next-line
+if (Cypress.env('RETRY_FAILED_TESTS')) {
```

Comment:
I'm 👍 for this, but I'm curious what the reasoning was. Are some of the tests flakey?

Reply:
Flakeys language bar and responsive nav bar tests.